components/net/fetch/methods.rs

```diff
@@ -498,3 +530,94 @@ fn is_null_body_status(status: &Option<StatusCode>) -> bool {
         _ => false
     }
 }
+
+/// https://fetch.spec.whatwg.org/#should-response-to-request-be-blocked-due-to-nosniff?
+#[inline]
```

Comment:
Let's remove these inline annotations - they only matter for code that is performance-sensitive, and this code is not.